src/__internal__/fieldset/fieldset.component.js

```diff
@@ -25,6 +25,7 @@ const Fieldset = ({
             width={ legendWidth }
             align={ legendAlign }
             rightPadding={ legendSpacing }
+            isRequired={ required || rest['aria-required'] }
```

Comment:
```suggestion
            isRequired={ isRequired}
```

This component doesn't need to support both props, we can simplify this.

Reply:
Done